sanic/exceptions.py

```diff
@@ -139,9 +139,12 @@ class PayloadTooLarge(SanicException):
 
 class Handler:
     handlers = None
+    cached_handlers = None
+    _missing = object()
```

Comment:
It actually means missing value. The value `_missing` means the given key does not exist in the cache (cache miss). For the default handler, the value `None` is saved in the cache.